.travis.yml

```diff
@@ -5,58 +5,42 @@ language: ruby
 
 cache: bundler
 
-before_install:
-  - gem update --system
-  - gem install bundler
+script:
+  - bundle exec rspec
```

Comment:
I believe `rubocop` is no run in Travis. It should be.

Reply:
Fixed.